randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -254,7 +250,9 @@ Extra - map_name: rm_a5c06
   * Normal Door to Meat Boy Room/Door to Energy Distribution Activation Shaft
   * Extra - instance_id: 133682
   > Door to Energy Distribution Activation
-      Space Jump Wall
+      Any of the following:
+          Space Jump Wall
+          Speed Booster and Shinesparking Tricks (Beginner) and Disabled Entrance Rando
```

Comment:
Seeing this makes me wonder whether they should have disabled door rando.
Since the way rdv thinks about it, is that if you *are* at the bottom door (and entrance rando is disabled), you would be able to shinespark up. But you can get to the bottom door, by just falling down from the top. Would also mean, that this is now likelier to be a dangerous resource.

Reply:
Interesting point, I'm not sure how to handle this. I'll pose the question to dread maintainers.